include/wil/win32_helpers.h

```diff
@@ -41,10 +41,10 @@
 #include "wistd_type_traits.h"
 
 /// @cond
-#if _HAS_CXX20 && defined(_STRING_VIEW_) && defined(_COMPARE_)
+#if (__WI_LIBCPP_STD_VER >= 20) && defined(_STRING_VIEW_) && defined(_COMPARE_)

```

Comment:
This needs some cleanup to ideally include `<compare>` when available and usable and kick the C++ standard version check, but that's for a future change.